python/ray/data/_internal/arrow_ops/transform_pyarrow.py

```diff
@@ -597,24 +597,31 @@ def try_combine_chunked_columns(table: "pyarrow.Table") -> "pyarrow.Table":
     return pyarrow.Table.from_arrays(new_column_values_arrays, schema=table.schema)
 
 
-def combine_chunks(table: "pyarrow.Table", copy: bool = False) -> "pyarrow.Table":
+def combine_chunks(
+    table: "pyarrow.Table", always_copy: bool = False
+) -> "pyarrow.Table":
     """This is counterpart for Pyarrow's `Table.combine_chunks` that's using
     extended `ChunkedArray` combination protocol.
 
     For more details check out `combine_chunked_array` py-doc
+
+    Args:
+        table: Table with chunked columns to be combined into contiguous arrays.
+        always_copy: Skip copying when always_copy is False and there is exactly
+        1 chunk.
     """
 
     new_column_values_arrays = []
 
     for col in table.columns:
-        new_column_values_arrays.append(combine_chunked_array(col, copy))
+        new_column_values_arrays.append(combine_chunked_array(col, always_copy))
 
     return pyarrow.Table.from_arrays(new_column_values_arrays, schema=table.schema)
 
 
 def combine_chunked_array(
     array: "pyarrow.ChunkedArray",
-    copy: bool = False,
+    always_copy: bool = False,
```

Comment:
Let's keep it as `copy` -- this rename make it more confusing adding a dimension you now need to think about as to what exact scenarios "always" is referring to